### PR TITLE
[FIX] German translations 

### PR DESCRIPTION
**Pull request by kaiiiiiiiii**
@RocketChat/ios

This PR revises the German translation and brings it into a formal style to keep it in line with the server translation.

### CODE REVIEW

File: Rocket.Chat/Resources/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -7,122 +7,122 @@
...
+"auth.login.buttonResetPassword" = "Passwort vergessen";
+"auth.login.agree_label" = "Indem Sie fortfahren, erklären Sie sich einverstanden mit unserer";
+"auth.login.agree_and" = "und";
+"auth.login.agree_termsofservice" = "Allgemeine Geschäftsbedingungen";
+"auth.login.agree_privacypolicy" = "Datenschutzerklärung";
 
 "auth.forgot_password.title" = "Passwort vergessen?";
-"auth.forgot_password.message" = "E-Mail-Adresse eingeben";
-"auth.forgot_password.success" = "Wenn diese E-Mail-Adresse hinterlegt ist, werden Anweisungen zum Zurücksetzen des Passworts verschickt. Wenn du keine Mail bekommen hast, versuche es noch einmal.";
+"auth.forgot_password.message" = "Bitte geben Sie Ihre E-Mail-Adresse eingeben";
```
Comment (ArchimedesFM) [inline Rocket.Chat/Resources/de.lproj/Localizable.strings]: Bitte geben Sie Ihre E-Mail-Adresse ein

File: Rocket.Chat/Resources/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -7,122 +7,122 @@
...
-"auth.login.agree_and" = "and";
-"auth.login.agree_termsofservice" = "Terms of Service";
-"auth.login.agree_privacypolicy" = "Privacy Policy";
+"auth.login.username.placeholder" = "E-Mail oder Benutzername";
+"auth.login.password.placeholder" = "Passwort";
+"auth.login.buttonRegister" = "Neues Konto erstellen";
+"auth.login.buttonResetPassword" = "Passwort vergessen";
+"auth.login.agree_label" = "Indem Sie fortfahren, erklären Sie sich einverstanden mit unserer";
+"auth.login.agree_and" = "und";
+"auth.login.agree_termsofservice" = "Allgemeine Geschäftsbedingungen";
```
Comment (ArchimedesFM) [inline Rocket.Chat/Resources/de.lproj/Localizable.strings]: Allgemeinen Geschäftsbedingung?

**Comment by kaiiiiiiiii** [comment]
Thanks for reviewing @ArchimedesFM :)

**Comment by rafaelks** [comment]
Looking great @kaiiiiiiiii, thank you very much! :-)

**Pull request merged**